site/docs/v1/tech/lambdas/_saml-response-fields.adoc

```diff
@@ -21,11 +21,10 @@ The instant that this assertion stops being valid. This is the number of millise
 [field]#samlResponse.assertion.issuer# [type]#[String]#::
 The issuer of this SAML assertion. This defaults to the `issuser` from the corresponding SAML identity provider configuration.
 
-[field]#samlResponse.assertion.subject.nameID.format# [type]#[String]#::
-The `NameID` format for the id of the subject (user). FusionAuth uses the `emailaddress` format specified by the SAML specifications. It is not recommended that you change this unless you know what you are doing.
-
-[field]#samlResponse.assertion.subject.nameID.id# [type]#[String]#::
-The `NameID` id of the subject (user). This defaults to the user's email address. It is not recommended that you change this unless you know what you are doing.
+[field]#samlResponse.assertion.subject.nameIDs# [type]#[Array<Object>]#::
```

Comment:
We normally document arrays of objects as separate fields. See User API as an example. 

For example:

```
[field]#samlResponse.assertion.subject.nameIDs# [type]#[Array]#::

[field]#samlResponse.assertion.subject.nameIDs``[x]``.format# [type]#[String]#::

[field]#samlResponse.assertion.subject.nameIDs``[x]``.id# [type]#[String]#::
```

If we are going to document the default value that we set, we should probably use the FQ name which is `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`.